TM1py/Services/CellService.py

```diff
@@ -469,19 +534,25 @@ def write_through_unbound_process(self, cube_name: str, cellset_as_dict: Dict, i
                     name="",
                     prolog_procedure=enable_sandbox + "\r".join(chunk[:Process.MAX_STATEMENTS]),
                     epilog_procedure="\r".join(chunk[Process.MAX_STATEMENTS:]))
-                success, _, _ = self.execute_unbound_process(process, **kwargs)
+                success, status,error_log_file = self.execute_unbound_process(process, **kwargs)
                 successes.append(success)
+                if error_log_file:
+                    message = self.get_error_log_file_content(file_name=error_log_file)
+                    messages.append(message)
+
                 chunk = list()
 
         process = Process(
             name="",
             prolog_procedure=enable_sandbox + "\r".join(chunk[:Process.MAX_STATEMENTS]),
             epilog_procedure="\r".join(chunk[Process.MAX_STATEMENTS:]))
-        success, _, _ = self.execute_unbound_process(process, **kwargs)
+        success, status, error_log_file = self.execute_unbound_process(process, **kwargs)
         successes.append(success)
+        if error_log_file:
+            message = self.get_error_log_file_content(file_name=error_log_file)
+            messages.append(message)
 
-        if not all(successes):
-            raise TM1pyException(f"{len(successes) - sum(successes)} out of {len(successes)} unbound processes failed")
+        return all(successes), messages, changeset
```

Comment:
I imagine retrieving all error_log_files may slow down the function.
Perhaps you could just return the name of the error_log_files and leave it to the caller to read the details and understand what went wrong in detail.

Reply:
We can pass back a list of log files instead. I was trying to bring message parity to the different write methods. Messages from MDX updates don't need to be "unpacked"

Reply:
OK I see. I would prefer to just return the log_file_names to not lose performance.